htdocs/projet/element.php

```diff
@@ -397,8 +397,8 @@
 	'urlnew'=>DOL_URL_ROOT.'/commande/card.php?action=create&projectid='.$id.'&socid='.$socid.'&backtopage='.urlencode($_SERVER['PHP_SELF'].'?id='.$id),
 	'lang'=>'orders',
 	'buttonnew'=>'CreateOrder',
-	'testnew'=>$user->rights->commande->creer,
-	'test'=>$conf->commande->enabled && $user->rights->commande->lire),
+	'testnew'=>!empty($user->rights->commande->creer),
```

Comment:
The good way to fix warning on permission is to use the method  hasRight() so
`'testnew'=>$user->hasRight('commande', 'creer'),`